Kernel/Devices/HID/PS2/MouseDevice.cpp

```diff
@@ -1,39 +1,30 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <Kernel/Arch/x86_64/Hypervisor/VMWareBackdoor.h>
-#include <Kernel/Arch/x86_64/ISABus/HID/PS2MouseDevice.h>
+#include <Kernel/Arch/x86_64/ISABus/I8042Controller.h>
```

Comment:
This is breaking the build for aarch64.

Reply:
Right, I didn't test aarch64, and now that you say this will break this target, I will look into making it more abstracted...